### ISSUE TITLE
Metamask cannot be connected

### ISSUE THREAD

**Issue by CaptnVegas** [soonaverse/soonaverse-dao#824]
**Describe the bug**
Initially I tried to make a change to a NFT that I currently have for sale. However only the spinny rotating orange thing comes up on Sonnaverse and metamask never opened to sign anything. This only occurs on my brave browser. I switched to firefox and all works fine there. In an attempt to fix it I disconnected MM from Sonnaverse in my brave browser. Now I cannot connect MM anymore. Same issue, spinny orange thing but not request coming up on MM.

**To Reproduce**
1. Click connect on the top right on soona
2. Click on MM
3. Spinny screen comes up
4. MM does not open

**Expected behavior**
MM should open and I should be able to sign

**Screenshots**
![image](https://user-images.githubusercontent.com/110666203/183052779-afe227ed-b9ae-44c1-a70d-6e18716f0be0.png)

**Desktop (please complete the following information):**
 - OS: [Win 10]
 - Browser [Brave]
 - Version [1.42.86 (newest)]

**Additional context**
MM in my brave is btw working fine on other sites like Uniswap.
Upon research I saw that in the past some people seem to have issues like this and fixed it by turning of the Brave Wallet in the Brave settings. This did not help me here.

**Comment by adamunchained** [comment]
Do you use any extensions? Did you try to disable them?

**Comment by CaptnVegas** [comment]
Hi Adam, 
I do have 3 additional extensions. Disabling however did not help. 
However I just found a resolution. First I tried to delete only cache + images and the cookies of soonaverse which did not help.
Deleting the **complete** browser history / cache images and cookies did the trick. After deleting everything (basicly resetting the whole browser) MM is now popping up again.
![image](https://user-images.githubusercontent.com/110666203/183240923-b8eeca24-e0f5-4970-b106-525c355354d3.png)

Thanks CaptainVegas
